include/zephyr/bluetooth/classic/avdtp.h

```diff
@@ -4,49 +4,154 @@
 
 /*
  * Copyright (c) 2015-2016 Intel Corporation
+ * Copyright 2024 NXP
  *
  * SPDX-License-Identifier: Apache-2.0
  */
 #ifndef ZEPHYR_INCLUDE_BLUETOOTH_AVDTP_H_
 #define ZEPHYR_INCLUDE_BLUETOOTH_AVDTP_H_
 
-#include <zephyr/bluetooth/l2cap.h>
-
 #ifdef __cplusplus
 extern "C" {
 #endif
 
-/** @brief AVDTP SEID Information */
+/**
+ * @brief AVDTP error code
+ */
+enum bt_avdtp_err_code {
+	/** The response is success, it is not from avdtp spec. */
+	BT_AVDTP_SUCCESS = 0x00,
+	/** The request is time out without response, it is not from avdtp spec. */
+	BT_AVDTP_TIME_OUT = 0xFF,
+	/** The request packet header format error */
+	BT_AVDTP_BAD_HEADER_FORMAT = 0x01,
+	/** The request packet length is not match the assumed length */
+	BT_AVDTP_BAD_LENGTH = 0x11,
+	/** The requested command indicates an invalid ACP SEID (not addressable) */
+	BT_AVDTP_BAD_ACP_SEID = 0x12,
+	/** The SEP is in use */
+	BT_AVDTP_SEP_IN_USE = 0x13,
+	/** The SEP is not in use */
+	BT_AVDTP_SEP_NOT_IN_USE = 0x14,
+	/** The value of Service Category in the request packet is not defined in AVDTP */
+	BT_AVDTP_BAD_SERV_CATEGORY = 0x17,
+	/** The requested command has an incorrect payload format */
+	BT_AVDTP_BAD_PAYLOAD_FORMAT = 0x18,
+	/** The requested command is not supported by the device */
+	BT_AVDTP_NOT_SUPPORTED_COMMAND = 0x19,
+	/** The reconfigure command is an attempt to reconfigure a transport service codec_cap
+	 * of the SEP. Reconfigure is only permitted for application service codec_cap
+	 */
+	BT_AVDTP_INVALID_CAPABILITIES = 0x1A,
+	/** The requested Recovery Type is not defined in AVDTP */
+	BT_AVDTP_BAD_RECOVERY_TYPE = 0x22,
+	/** The format of Media Transport Capability is not correct */
+	BT_AVDTP_BAD_MEDIA_TRANSPORT_FORMAT = 0x23,
+	/** The format of Recovery Service Capability is not correct */
+	BT_AVDTP_BAD_RECOVERY_FORMAT = 0x25,
+	/** The format of Header Compression Service Capability is not correct */
+	BT_AVDTP_BAD_ROHC_FORMAT = 0x26,
+	/** The format of Content Protection Service Capability is not correct */
+	BT_AVDTP_BAD_CP_FORMAT = 0x27,
+	/** The format of Multiplexing Service Capability is not correct */
+	BT_AVDTP_BAD_MULTIPLEXING_FORMAT = 0x28,
+	/** Configuration not supported */
+	BT_AVDTP_UNSUPPORTED_CONFIGURAION = 0x29,
+	/** Indicates that the ACP state machine is in an invalid state in order to process the
+	 * signal. This also includes the situation when an INT receives a request for the
+	 * same command that it is currently expecting a response
+	 */
+	BT_AVDTP_BAD_STATE = 0x31,
+};
+
+/** @brief Stream End Point Role */
+enum bt_avdtp_role_type {
```

Comment:
I'd drop the `_type` from the struct name

Reply:
Updated.